wandb/integration/yolov8/yolov8.py

```diff
@@ -0,0 +1,267 @@
+from typing import List, Optional
```

Comment:
could you add type annotations (for example: return values)
you can turn the checks on in the mypy.ini file

Reply:
:100: Added type annotations for return values of all methods and functions.